docs/bis/ec/update-koq-on-property.md

```diff
@@ -0,0 +1,186 @@
+				<From>u:CM</From>
+				<To>u:M</To>
+			</AllowUnitChange>
+		</ECCustomAttributes>
+	</ECProperty>
+</ECEntityClass>
+```
+
+#### Invalid/malformed `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md)
+
+```xml
+<KindOfQuantity typeName='KindOfQuantity1' description='KindOfQuantity1'
+				displayLabel='KindOfQuantity1' persistenceUnit='u:CM' relativeError='.5'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<KindOfQuantity typeName='KindOfQuantity2' description='KindOfQuantity2'
+				displayLabel='KindOfQuantity2' persistenceUnit='u:M' relativeError='.2'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='KindOfQuantity2' >
+	</ECProperty>
+</ECEntityClass>
+```
+
+```xml
+<KindOfQuantity typeName='KindOfQuantity1' description='KindOfQuantity1'
+				displayLabel='KindOfQuantity1' persistenceUnit='u:CM' relativeError='.5'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<KindOfQuantity typeName='KindOfQuantity2' description='KindOfQuantity2'
+				displayLabel='KindOfQuantity2' persistenceUnit='u:M' relativeError='.2'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='KindOfQuantity2' >
+		<ECCustomAttributes>
+			<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			</AllowUnitChange>
+		</ECCustomAttributes>
+	</ECProperty>
+</ECEntityClass>
+```
+
+```xml
+<KindOfQuantity typeName='KindOfQuantity1' description='KindOfQuantity1'
+				displayLabel='KindOfQuantity1' persistenceUnit='u:CM' relativeError='.5'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<KindOfQuantity typeName='KindOfQuantity2' description='KindOfQuantity2'
+				displayLabel='KindOfQuantity2' persistenceUnit='u:M' relativeError='.2'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='KindOfQuantity2' >
+		<ECCustomAttributes>
+			<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+				<From>u:CM</From>
+			</AllowUnitChange>
+		</ECCustomAttributes>
+	</ECProperty>
+</ECEntityClass>
+```
+
+```xml
+<KindOfQuantity typeName='KindOfQuantity1' description='KindOfQuantity1'
+				displayLabel='KindOfQuantity1' persistenceUnit='u:CM' relativeError='.5'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<KindOfQuantity typeName='KindOfQuantity2' description='KindOfQuantity2'
+				displayLabel='KindOfQuantity2' persistenceUnit='u:M' relativeError='.2'
+				presentationUnits='f:AmerFI[u:FT][u:IN]' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='KindOfQuantity2' >
+		<ECCustomAttributes>
+			<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+				<From>u:CM</From>
+				<To></To>
+			</AllowUnitChange>
+		</ECCustomAttributes>
+	</ECProperty>
+</ECEntityClass>
+```
```

Comment:
Same here, no need to explicitly show all invalid options ... just list what the requirements are